EIPS/eip-6609.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 6609
+title: A parallel system that runs next to the EVM.
+description: A gasless system that runs next to the EVM that uses reputation (rep) instead of gas.
+author: JKincorporated (@JKincorporated)
+discussions-to: https://ethereum-magicians.org/t/draft-eip-parallel-system-that-runs-next-to-the-evm/13155
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-03-03
+---
+
+## Abstract
+
+This EIP proposes a gasless system for that runs next to the EVM. This system has no native currency so all currencies should be made using smart contracts. The proposed system aims to prevent gas fees while maintaining a secure system.
+  
+This is still a work in progress and no where near close enough for production.
+
+
+## Motivation
+
+The existing Ethereum gas system can be a barrier to entry for some users, especially those who cannot afford to pay the high gas fees. A gasless system based on reputation can enable more users to participate in the Ethereum ecosystem. Additionally, the proposed system aims to prevent multiple accounts and limit execution of transactions to users with a minimum reputation score to reduce the risk of fraudulent activities.
+
+## Specification
+
+### PBEVM (Python Based Ethereum Virtual Machine)
+
+Instead of the EVM this uses the PBEVM (Python Based Ethereum Virtal Machine), This is because having 2 EVMs sounds weird.
+As the name suggests all smart contracts that are to be run on this VM should be writting in Python.
```

Comment:
I think you are confusing languages with bytecode here. You're suggesting that the code should execute in python verbatim? If so, there are a few issues with that:

* Python is horrendously inefficient
* Python isn't strongly-typed
* You haven't specified how:
  * Gas is consumed
  * Calls are made
  * The execution layer knows what "VM" the contract is using
* Python has no formal specification
* EL clients not written in python would have trouble implementing this.

And the biggest issue of all:

* **Your proposal can be implemented with a python module and operator overloading without requiring a hard fork!**

As such, I will not be approving this EIP *unless* you can provide a strong motivation for it and rewrite it such that it can be implemented and makes sense to implement.

Reply:
You make a lot of valid points, Should I scrap this EIP and just focus on the reputation system?

Reply:
Yes, I would recommend that.

Reply:
I just close the PR to close right?